act/utils/io_utils.py

```diff
@@ -132,7 +132,7 @@ def unpack_tar(
     for tar_file in tar_files:
         try:
             tar = tarfile.open(tar_file)
-            tar.extractall(path=out_dir)
+            tar.extractall(path=out_dir, filter='data')
```

Comment:
Should we add this in as a keyword option as well in case users run into some issues?